service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ManageDocumentsControllerAboutToSubmitTest.java

```diff
@@ -65,7 +64,7 @@ void shouldPopulateHearingFurtherDocumentsCollection() {
                     .code(hearingId)
                     .build())
                 .build(),
-            TEMP_FURTHER_EVIDENCE_DOCUMENTS_COLLECTION_KEY, furtherEvidenceBundle,
+            TEMP_EVIDENCE_DOCUMENTS_COLLECTION_KEY, furtherEvidenceBundle,
```

Comment:
You said i should removed this in my PR. Is that right? Are you waiting for my PR to go first?

Reply:
CaseField has changed so in my PR it is needed.